test/witan/gateway/integration/base.clj

```diff
@@ -13,10 +15,11 @@
 
 (defn wait-for-pred
   ([p]
-   (wait-for-pred p 100))
+   (wait-for-pred p (or (env :wait-tries) 65)))
```

Comment:
This will annoy you, but env always returns strings. So, you want to do (def wait-tries (Integer/valueOf (env :wait-tries "65"))). Otherwise you'll get tears when you try to use it.